renderdoc/driver/metal/metal_device.cpp

```diff
@@ -471,6 +472,205 @@ WrappedMTLRenderPipelineState *WrappedMTLDevice::newRenderPipelineStateWithDescr
   return wrappedMTLRenderPipelineState;
 }
 
+template <typename SerialiserType>
+bool WrappedMTLDevice::Serialise_newComputePipelineStateWithDescriptor(
+    SerialiserType &ser, WrappedMTLComputePipelineState *pipelineState,
+    RDMTL::ComputePipelineDescriptor &descriptor, MTL::PipelineOption options,
+    MTL::AutoreleasedComputePipelineReflection *reflection, NS::Error **error)
+{
+  SERIALISE_ELEMENT_LOCAL(ComputePipelineState, GetResID(pipelineState))
+      .TypedAs("MTLComputePipelineState"_lit);
+  SERIALISE_ELEMENT(descriptor);
+
```

Comment:
I think "options" should be being serialised here. 
The reflection information might not be wanted or need to be serialised and recreated during replay.
Options has "MTLPipelineOptionFailOnBinaryArchiveMiss" which might be important to handle during replay.